### ISSUE TITLE
[css-lists] Does counter() create a new counter if none exists?

### ISSUE THREAD

**Issue by Loirooriol** [w3c/csswg-drafts#2348]
Consider this code: https://jsfiddle.net/35kuzg5m/

```html
<div>
  <p>aaa</p>
  <p>bbb</p>
</div>
<hr />
<div>
  <p>aaa</p>
  <p>bbb</p>
</div>
```

```css
p::before {
  content: counter(c) '/ ';
  counter-increment: c 1;
}
div ~ div::before {
  content: counter(c, none);
}
```

It's rendered as:

| Firefox and Chrome  | Edge                |
| ------------------- | ------------------- |
| 1/ aaa<br />1/ bbb  | 1/ aaa<br />1/ bbb  |
| 1/ aaa<br />2/ bbb  | 1/ aaa<br />1/ bbb  |

In https://drafts.csswg.org/css-lists/#counter-functions I can't find anything that says that `counter()` can [create a new counter](https://drafts.csswg.org/css-lists/#create-a-new-counter), so it seems Edge is the only one that follows it.

**Comment by tabatkins** [comment]
Right, that sort of behavior is definitely not documented. I wasn't aware that it happened, or even remotely believed that it could. ^_^

**Comment by Loirooriol** [comment]
Oh, it seems Firefox and Chrome follow [CSS 2.2](https://www.w3.org/TR/CSS22/generate.html#x29):

> If `counter-increment` or `content` on an element or pseudo-element refers to a counter that is not in the scope of any `counter-reset`, implementations should behave as though a `counter-reset` had reset the counter to 0 on that element or pseudo-element.

**Comment by tabatkins** [comment]
Sigh, cool, I must have missed that when I was drafting the Lists spec.

**Comment by fantasai** [comment]
Fixed see 09f1d2bfa93183b0c8fe2b54438f850a383ff0a9 and 848b8c611bffea756866a53972ce531b8a649e4d and maybe a few changesets in between

**Comment by fantasai** [comment]
Marked Obvious Bugfix since we're supposed to be in sync with CSS2.1
